cmd/liqoctl/cmd/offload.go

```diff
@@ -110,6 +111,7 @@ func newOffloadNamespaceCommand(ctx context.Context, f *factory.Factory) *cobra.
 		"The constraints regarding pods scheduling in this namespace, among Local, Remote and LocalAndRemote")
 	cmd.Flags().Var(namespaceMappingStrategy, "namespace-mapping-strategy",
 		"The naming strategy adopted for the creation of remote namespaces, among DefaultName and EnforceSameName")
+	cmd.Flags().DurationVar(&options.Timeout, "timeout", 120*time.Second, "The timeout for the offloading process")
```

Comment:
Would lower the timeout to something in the range 10-30 seconds, as more than enough to complete the process. Unoffloading, insteat, might take longer, since namespace deletion is typically slow.